src/devices/src/virtio/vsock/csm/txbuf.rs

```diff
@@ -50,17 +49,9 @@ impl TxBuf {
             return Err(Error::TxBufFull);
         }
 
-        // We're using a closure here to return the boxed slice, instead of a value (i.e.
```

Comment:
nit: Why did you remove this comment?

Reply:
It was outdated. Clippy got fixed in the meantime, and we'd removed the clippy bypass, but forgot to also removed the comment.